src/xbot/common/command/SupportsSetpoint.java

```diff
@@ -0,0 +1,8 @@
+package xbot.common.command;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public interface SupportsSetpoint {
```

Comment:
this is more specific than maybe SupportsSetPoint name implies, maybe `SupportsSetpointSubsystem`? Like I know wpi has it's own similar interface